CONTRIBUTING.md

```diff
@@ -167,50 +204,104 @@ make license-check
+Start the backporting process by making sure both the `main` and the target version branch are up-to-date in your working copy. Look up the commit ID of the commit and then switch to your target version branch. Now create a new branch from that version branch:
 
+```sh
+git checkout -b backports/ISSUE_NUMBER
 ```
-TF_ACC=1 go test ./internal/initwd
+
+Now you can cherry-pick the commit in question to your backport branch:
+
+```sh
+git cherry-pick -s COMMIT_ID_HERE
 ```
 
-Because the acceptance tests depend on services outside of the OpenTofu codebase, and because the acceptance tests are usually used only when making changes to the systems they cover, it is common and expected that drift in those external systems will cause test failures. Because of this, prior to working on a system covered by acceptance tests it's important to run the existing tests for that system in an *unchanged* work tree first and respond to any test failures that preexist, to avoid misinterpreting such failures as bugs in your new changes.
+Finally, create an additional commit to edit the changelog and send in your PR. 
 
-### Integration Tests: Testing interactions with external backends
+---
 
-OpenTofu supports various [backends](https://opentofu.org/docs/language/settings/backends/configuration). We run integration test against them to ensure no side effects when using OpenTofu.
+## FAQ
 
-Execute to list all available commands to run tests:
+### Who decides if a feature will be implemented and how is that decision made?
 
-```commandline
-make list-integration-tests
-```
+When you submit an enhancement request, the [core team](#who-is-the-core-team) looks at your issue first. Given the size of the code, adding a new feature is always a careful balancing act. The core team takes the following points into consideration:
 
-From the list of output commands, you can execute those which involve backends you intend to test against.
+1. **Is it possible to implement this on a technical level?**<br />Sometimes, even if a feature would be extremely useful, the state of the codebase doesn't let us do it. 
+2. **Does the feature cause more technical debt?**<br />A feature request may hide a larger issue under the hood. Sometimes it is more desirable to resolve the underlying issue instead of implementing the feature in isolation.
+3. **Is there someone who would do the work?**<br />The core team doesn't have the capacity to implement everything, so for many issues community contributions are very welcome. Sometimes companies external to OpenTofu decide to dedicate developers for the development of a specific larger feature, which can also weigh in on the decision-making process.
+4. **Is there enough capacity on the core team to support a community contributor?**<br />We don't expect contributors to implement the entire feature in isolation, we actively participate with planning, reviews and writing documentation as needed.
```

Comment:
I would clarify in the description here that:

"We don't expect contributors to implement the entire feature in isolation, we actively participate with planning, reviews and writing documentation as needed. Thus, core engineers need to be available in order for an issue to be tackled, even if it's being tackled by the community."

Rephrase as needed, I don't think my version reads too nice.